src/main/java/pl/nextleveldev/smart_route/model/compositePK/BusStopLineId.java

```diff
@@ -0,0 +1,40 @@
+package pl.nextleveldev.smart_route.model.compositePK;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+
+@Embeddable
+@Getter
+@Setter
+public class BusStopLineId implements Serializable {
+
+    @Column(name = "line_id")
+    private UUID lineId;
+
+    @Column(name = "bus_stop_id")
+    private UUID busStopId;
+
+    public BusStopLineId() {}
+
+    public BusStopLineId(UUID lineId, UUID busStopId) {
+        this.lineId = lineId;
+        this.busStopId = busStopId;
+    }
+
+    @Override
```

Comment:
```suggestion
@Override
public boolean equals(Object o) {
    if (this == o) { return true; }
    if (!(o instanceof BusStopLineId that)) { return false; }
    return Objects.equals(lineId, that.lineId) &&
           Objects.equals(busStopId, that.busStopId);
}
```

Using curly braces for single statements does not affect the code's operation, but may improve readability and make future modifications easier. And by the way, there will be no reports in static code analysis (SonerQube, Checkstyle, etc..)